lib/api/controllers/authController.js

```diff
@@ -79,8 +79,9 @@ class AuthController {
     const strategy = request.input.args.strategy;
 
     return this.kuzzle.passport.authenticate({body: request.input.body, query: request.input.args, original: request}, strategy)
+      .then(userObject => this.kuzzle.pluginsManager.trigger('passport:afterAuthenticate', {strategy, user:userObject}))
```

Comment:
we create a JSON object with the strategy the user object, to enable the pipe listener to handle different stuff according to the actual strategy if needed.